config/default/plugin-site.yaml

```diff
@@ -0,0 +1,33 @@
+ingress:
+  enabled: true
+  annotations:
+    "certmanager.k8s.io/cluster-issuer": "letsencrypt-prod"
+    "kubernetes.io/ingress.class": "public-ingress"
+    "nginx.ingress.kubernetes.io/rewrite-target": /$1
+    "nginx.ingress.kubernetes.io/hsts": "true"
+    "nginx.ingress.kubernetes.io/ssl-redirect": "true"
+
+  hosts:
+    - host: plugins.jenkins.io
+      paths:
+        - path: /api/(.*)
+          port: backend
+        - path: /(.*)
+          port: frontend
+
+  tls:
+    - secretName: plugins-tls
+      hosts:
+        - plugins.jenkins.io
+
+restApiUrl: https://plugins.jenkins.io/api
+
+replicaCount: 2
+
+resources:
+  limits:
+    cpu: 1500m
+    memory: 2048Mi
```

Comment:
This value is definitely too high is more than probably too high and we'll adjust it once we have more infromation from the monitoring

Reply:
Sure, I didn't want to be too conservative on the limits, but needs monitoring data 👍